src/model.js

```diff
@@ -12,8 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-const { Schema } = require('mongoose');
-const mongoose = require('mongoose');
+const { Schema } = require('mongoose')
```

Comment:
We should not remove ;

Reply:
All semicolon removals should now be reverted